crates/stc_ts_file_analyzer/tests/visualize/exprs/bin/comparison/null-1.swc-stderr

```diff
@@ -69,7 +69,7 @@ Error:
     `----
 
 Error: 
-  > (string | undefined)
+  > string
```

Comment:
I think this is a wrong change.

Reply:
Why did you remove `undefined` from the union?

Reply:
He is not strict